docs/sources/panels/value-mappings.md

```diff
@@ -3,14 +3,124 @@ title = "Value mappings"
 weight = 600
 +++
 
-## Value mappings
+# Value mappings
 
-Value mappings come in different types.
+Value mapping concept allow you to replace values or ranges in your visualizations with words or emojis.
 
-* **Range** maps numerical ranges to a display text and color. 
-* **Value** maps text values to a color or different display text.
-* **Special** maps special values like `Null`, `NaN` and boolean values like `true` and `false` to a display text and color.
+Values mapped via value mappings will skip the unit formatting. This means that a text value mapped to a numerical value will not be formatted using the configured unit.
 
-The display text and color are both optional. If you only want to assign colors to text values you can leave the display text empty and the original value will be used for display.
+![Value mappings example](/static/img/docs/value-mappings/value-mappings-example-8-0.png)
 
-Values mapped via value mappings will skip the unit formatting. This means that a text value mapped to a numerical value will not be formatted using the configured unit.
\ No newline at end of file
+If value mappings are present in a panel, then Grafana displays a summary in the side pane of the panel editor.
+
+## Value mapping examples
+
+Value mappings are displayed differently in different visualizations.
+
+### Time series example
+
```

Comment:
Will be nice to have an introductory/stem sentence.

Reply:
Added